documentation/api/query/curl.markdown

```diff
@@ -82,6 +82,17 @@ If you do this with an endpoint that accepts `GET` requests, **you must also use
     curl -G http://localhost:8080/pdb/query/v4/nodes \
       --data-urlencode 'query=["=", ["node", "active"], true]'
 
+## Pretty querying of PuppetDB
+
+PuppetDB returns unprettified JSON by default. PuppetDB provides the option of
+prettifying your JSON responses with the `pretty` parameter, setting it to any
+non-`null` value will work. Be aware pretty printing comes at the cost of
```

Comment:
The only kind of precedence I've seen for a param like this is GitHub and the `?w=<anything>` param.